src/tests/ftest/osa/osa_online_extend.py

```diff
@@ -204,35 +141,31 @@ def run_online_extend_test(self, num_pool):
             self.log.info("Extra Servers = %s", self.extra_servers)
             self.start_additional_servers(self.extra_servers)
             # Give sometime for the additional server to come up.
-            time.sleep(5)
+            time.sleep(25)
```

Comment:
25 secs is again a bit long sleep.
Can we instead find the state of the newly added server if it has started or not? I see couple of methods in server_utils `detect_io_server_start()` and `get_single_system_state`. may be we can use one of those or if not that we can surely use dmg system query for the newly added rank and check it's state.

Reply:
Good suggestion... I would like to create a new PR and will address this issue... With extend still blocked, let's wait for the actual dev changes before we can test this feature. I don't want to mess-up some thing with this PR... Already on my 24 CI run iteration... Really time consuming...